src/tests/EdFi.DataManagementService.Tests.E2E/Features/General/DiscoveryAPI.feature.disabled

```diff
@@ -0,0 +1,50 @@
+Feature: Discovery API that retrieves the information from the base endpoint of the API
```

Comment:
I would not say that this "retrieves the information" - it _provides_ information, not _retrieves_ it. For the feature line above, please change to "The Discovery API provides information about the application version, supported data model(s), and URLs for additional metadata."

Reply:
Fixed